tests/functional/tests/conftest.py

```diff
@@ -3,18 +3,19 @@
 # SPDX-License-Identifier: BSD-3-Clause-Clear
 #
 
-import os
+
 import sys
+import os
 import pytest
 
 sys.path.append(os.path.join(os.path.dirname(__file__), os.path.pardir))
+
 from pyocf.types.logger import LogLevel, DefaultLogger, BufferLogger
 from pyocf.types.volume import Volume, ErrorDevice
 from pyocf.types.ctx import get_default_ctx
-
-
-def pytest_configure(config):
```

Comment:
Doesn't seem like a good idea to remove that, if someone hasn't got pyocf on env PATH/PYTHONPATH the tests won't find it.